lib/libs/email/content/upload-subsequent-documents/emailTemplates/MedSpaState.tsx

```diff
@@ -0,0 +1,39 @@
+import { Events } from "shared-types";
+import { CommonEmailVariables } from "shared-types";
+import { Text } from "@react-email/components";
+import { PackageDetails, DetailsHeading, Attachments, BasicFooter } from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
+import { styles } from "../../email-styles";
+
+export const MedSpaStateEmail = (props: {
+  variables: Events["UploadSubsequentDocuments"] & CommonEmailVariables;
+}) => {
+  const variables = props.variables;
+  const previewText = `Additional documents submitted for Medicaid SPA ${variables.id}`;
+  const heading = `You’ve successfully submitted the following to CMS reviewers for Medicaid SPA ${variables.id}`;
+
+  return (
+    <BaseEmailTemplate
+      previewText={previewText}
+      heading={heading}
+      applicationEndpointUrl={variables.applicationEndpointUrl}
+      footerContent={<BasicFooter />}
+    >
+      <DetailsHeading />
```

Comment:
Also could you remove the <DetailsHeading /> as the confluence doesn't have this text